reactivesocket-cpp/test/tcp/TcpServer.cpp

```diff
@@ -0,0 +1,141 @@
+//
+// Created by Yuri Schimke on 6/20/16.
+//
+
+
+
+#include <gmock/gmock.h>
+#include "reactivesocket-cpp/src/tcp/TcpDuplexConnection.h"
+#include <thread>
+#include <folly/Memory.h>
+#include <folly/io/async/AsyncServerSocket.h>
+#include <reactivesocket-cpp/src/mixins/MemoryMixin.h>
+#include <reactivesocket-cpp/test/simple/PrintSubscriber.h>
+#include <reactivesocket-cpp/src/ReactiveSocket.h>
+#include <reactivesocket-cpp/src/RequestHandler.h>
+#include <reactivesocket-cpp/test/simple/CancelSubscriber.h>
+#include <reactivesocket-cpp/test/simple/CancelSubscription.h>
+#include <reactivesocket-cpp/src/framed/FramedDuplexConnection.h>
+
+using namespace ::testing;
+using namespace ::reactivesocket;
+using namespace ::folly;
+
+DEFINE_string(address, "9898", "host:port to listen to");
+
+namespace {
+    class ServerSubscription
```

Comment:
formatting seems to be pretty inconsistent in this diff. I dont' think we are using class indentation in this project.